### DISCUSSION TITLE
blog/react-loading-state-pattern

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#183] [General]
# blog/react-loading-state-pattern

Easily manage react loading state with React Hot Toast and SWR custom hooks.

https://theodorusclarence.com/blog/react-loading-state-pattern

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Comment by Lonli-Lokli** [comment]
Don't think it will work well with multiple buttons on screen

**Reply by theodorusclarence** [reply]
you can disable the other buttons with the `isLoading` from the hook

https://theodorusclarence.com/blog/react-loading-state-pattern#accessing-loading-state
